### ISSUE TITLE
Move program-areas directory and refactor program-areas.html

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#6323]
### Overview
We need to move the program-areas directory and refactor program-areas.html so that 
- we can use the contents in the files in the program-areas directory for individual program area pages in the future, and 
- the Program Areas page [^1] doesn't break when the program-areas directory is moved

### Action Items
- [x] Find the `program-areas` directory under `_data/internal`
- [x] Move the `program-areas` directory so it is outside of the `_data` directory
- [x] Rename the `program-areas` directory to `_program-areas`
- [x] In `_config.yml`, find the `collections` variable. For more info on collections, see _Collections_ [^2] in the _Resources/Instructions_ section.
  - [x] Add the following:
    ```
      program-areas:
        output: false
    ```
- [x] In `pages/program-areas.html`, refactor the code so that it works to pull information from the files in the `_program-areas` directory
- [x] Using Docker, check your changes on the Program Areas page [^1] in desktop, tablet, and mobile views against the live page on the website
- [x] Once your pull request is merged, go to #3328 and
  - [x] Check off this issue under the _Dependency_ section
  - [x] If all the issues in the _Dependency_ section are checked off, move #3328 to the **New Issue Approval** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
[^1]: [Program Areas](https://www.hackforla.org/program-areas)
[^2]: [Collections](https://jekyllrb.com/docs/collections/)

**Comment by JessicaLucindaCheng** [comment]
# Issue Writing
- **Availability:** 6 hours
- **ETA:** Monday, February 26, 2024

**Comment by JessicaLucindaCheng** [comment]
# Issue Writing
- **Progress:** Done writing the issue. Once my blocker is addressed, I have another dev lead review it.
- **Blocker:** Besides the [Program Areas](https://www.hackforla.org/program-areas) page, is there anywhere else that would use the information in the files in [_data/internal/program-areas](https://github.com/hackforla/website/tree/gh-pages/_data/internal/program-areas)? **Ans from Bonnie:** No, it's only Program Areas page currently.
- **Availability:** TBD depends on answer to blocker
- **ETA:** TBD depends on answer to blocker

### Tasks
- [x] If only the Program Areas page will be impacted, this issue is ready for prioritization.

**Comment by luisitocanlas** [comment]
i. Availability: 2/27/24 ~ 3/02/24
ii. ETA: EOD 3/02/24
